packages/modules/pricing/src/migrations/Migration20241129094116.ts

```diff
@@ -1,6 +1,6 @@
 import { Migration } from "@mikro-orm/migrations"
 
-export class Migration20241127223829 extends Migration {
+export class Migration20241129094116 extends Migration {
```

Comment:
thought: We probably shouldn't rename migrations, as it would otherwise be re-applied no?